plugins/modules/packaging/os/pkgin.py

```diff
@@ -116,17 +121,21 @@
 
 
 import re
+from enum import IntEnum
```

Comment:
That unfortunately doesn't work with Python 2.

Reply:
Is it possible to install additional dependencies like enum34 (https://pypi.org/project/enum34/)?